plugins/modules/artisan.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Ralf Langebrake <ralf@langebrake.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: artisan
+
+short_description: Laravel command line interface
+
+version_added: 8.4.0
+
+description:
+    - >
+      Artisan is the command line interface included with Laravel, the
+      PHP web application framework for artisans. The module was heavily
+      inspired by M(community.general.composer), which is typically used together with artisan.
+
+extends_documentation_fragment:
+    - community.general.attributes
+
+attributes:
+    check_mode:
+        support: full
+    diff_mode:
+        support: none
+
+options:
+    working_dir:
```

Comment:
A bit of nit picking but maybe `project_path` would convey a slightly more accurate semantic.